lib/aspect/apiUsageFingerprintAspect.ts

```diff
@@ -38,49 +39,40 @@ async function getBuildFiles(p: Project): Promise<File[]> {
 
 export function createApiUsageFingerprintAspect(
     api: string,
-    apiDefinition: ApiDefinition): Aspect<string[]> {
+    apiDefinition: ApiDefinition): Aspect<Array<{directory: string, usedApis?: string[], error?: any}>> {
     return {
         name: `api-usage-${api}`,
         displayName: `Used API versions for ${api}`,
         extract: async (p, pli) => {
             const buildFiles = await getBuildFiles(p);
             if (buildFiles && buildFiles.length > 0) {
-                const fps = [];
+                const usedApisPerModule: Array<{directory: string, usedApis?: string[], error?: any}> = [];
```

Comment:
Is it possible to structure this so that projects with errors calculating api-usage are queryable?

Reply:
There is now a different FP for this when errors are found